public/app/core/hooks/useStoredTablePageSize.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
```

Comment:
Can this hook be used in other context besides pagination? If not the correct place for this would be together with the `Pagination` component. For example in `Pagination.hooks.ts` file.

Reply:
I guess technically it "could", although it isn't for now, so I'll put it next to Pagination.

Reply:
I agree on this, as we discussed: let's start by using utils/components locally and then move them to core if we see they're used in other places